counterpoll/main.py

```diff
@@ -123,6 +124,45 @@ def disable():
     port_info['FLEX_COUNTER_STATUS'] = DISABLE
     configdb.mod_entry("FLEX_COUNTER_TABLE", PORT_BUFFER_DROP, port_info)
 
+# Ingress PG drop packet stat
+@cli.group()
+@click.pass_context
+def pg_drop(ctx):
+    """  Ingress PG drop counter commands """
+    ctx.obj = swsssdk.ConfigDBConnector()
```

Comment:
>swsssdk [](start = 14, length = 7)

swsssdk is deprecately largely in sonic-utilities. Could you adapt the new `ConfigDBConnector` as in latest code?

Reply:
is it also deprecated on 202012? this PR should go to 202012 as well thus need to know if the change should be done on specific branch only
@ayurkiv-nvda  FYI